src/sentry/models/projecttemplate.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model, sane_repr
+
+
+@region_silo_only_model
+class ProjectTemplate(Model):
+    """
+    Identifies a project template that can be used to create new projects.
+
+    This model links the project template options to the organization that owns them.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    name = models.CharField(max_length=200)
+    organization = FlexibleForeignKey("sentry.Organization")
+    date_added = models.DateTimeField(default=timezone.now, null=True)
+    updated_at = models.DateTimeField(default=timezone.now, null=True)
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_projecttemplate"
```

Comment:
Does this need any unique together constraints like only 1 per org?

Reply:
In this case, nope, we want to support multiple templates in an org. The feature use case is an organization wants to have a default template for their python backend services and their js FE services.

Reply:
Should the names be unique per org?

Reply:
oh snap, i did not read that very well. yeah, they should likely be unique names per org. thanks for the shout Colleen and the catch Dan!